go.mod

```diff
@@ -3,10 +3,37 @@ module github.com/10gen/realm-cli
 go 1.14
 
 require (
+	4d63.com/gochecknoinits v0.0.0-20200108094044-eb73b47b9fc4 // indirect
+	dmitri.shuralyov.com/go/generated v0.0.0-20170818220700-b1254a446363 // indirect
+	github.com/AlecAivazis/survey/v2 v2.2.2
+	github.com/Netflix/go-expect v0.0.0-20180615182759-c93bf25de8e8
+	github.com/alecthomas/gocyclo v0.0.0-20150208221726-aa8f8b160214 // indirect
+	github.com/alexkohler/nakedret v1.0.0 // indirect
 	github.com/edaniels/golinters v0.0.3
```

Comment:
This all looks great, I am a fan of the framework built out here! One small thing we can do is update this dependency to use https://github.com/mongodb-forks/golinters . Before Eric left, he migrated his repos to this org

Reply:
Nice catch!  Filed that to https://jira.mongodb.org/browse/REALMC-7430 for the time being